docs/index.md

```diff
@@ -1,17 +1,18 @@
 ---
 layout: page
-title: AddressBook Level-3
+title: RC4HDB
 ---
 
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
-[![codecov](https://codecov.io/gh/se-edu/addressbook-level3/branch/master/graph/badge.svg)](https://codecov.io/gh/se-edu/addressbook-level3)
+[![Java CI](https://github.com/AY2223S1-CS2103T-W12-3/tp/actions/workflows/gradle.yml/badge.svg)](https://github.com/AY2223S1-CS2103T-W12-3/tp/actions/workflows/gradle.yml)
+[![codecov](https://codecov.io/gh/AY2223S1-CS2103T-W12-3/tp/branch/master/graph/badge.svg)](https://codecov.io/gh/AY2223S1-CS2103T-W12-3/tp)
 
 ![Ui](images/Ui.png)
 
-**AddressBook is a desktop application for managing your contact details.** While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
+This project is named `RC4 Housing Database` (`RC4HDB` for short). RC4HDB is designed to aid **NUS RC4 housing management staff** in their day-to-day workflow
+by providing a means to store and manipulate data specific to **RC4**.
```

Comment:
Do you think it would be better if we changed 'data' to something more specific? i.e. resident information etc.

Reply:
yep sounds good.